cli/execute/execute.go

```diff
@@ -0,0 +1,138 @@
+package execute
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/cli/arg"
+	"github.com/buildbuddy-io/buildbuddy/cli/log"
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/util/rexec"
```

Comment:
We have a single enterprise dep in cli -- let's remove it rather than adding another.